services/info/src/test/java/org/cryptimeleon/incentive/services/info/HttpApiTest.java

```diff
@@ -0,0 +1,89 @@
+package org.cryptimeleon.incentive.services.info;
+
+import org.cryptimeleon.incentive.crypto.model.IncentivePublicParameters;
+import org.cryptimeleon.incentive.crypto.model.keys.provider.ProviderPublicKey;
+import org.cryptimeleon.incentive.crypto.model.keys.provider.ProviderSecretKey;
+import org.cryptimeleon.math.serialization.converter.JSONConverter;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+public class HttpApiTest {
```

Comment:
purpose of this class?
maybe a comment would be nice

Reply:
I refactored the info service tests :)